src/current/_includes/releases/v23.2/v23.2.5.md

```diff
@@ -0,0 +1,65 @@
+## v23.2.5
+
+Release Date: May 7, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-5-sql-language-changes">SQL language changes</h3>
+
+- The new [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) `sql.stats.virtual_computed_columns.enabled` enables collection of table statistics on virtual computed columns. [#120923][#120923]
+- The new [session variable]({% link {{ page.version.version}}/session-variables.md %}) `optimizer_use_virtual_computed_column_stats` configures the [optimizer]({% link {{ page.version.version }}/make-queries-fast.md %}) to consider table statistics on virtual computed columns. [#121179][#121179]
+- The new `FORCE_INVERTED_INDEX` [hint]({% link {{ page.version.version }}/indexes.md %}#selection) configures the [optimizer]({% link {{ page.version.version }}/make-queries-fast.md %})  to prefer a query plan scan over any inverted index of the hinted table. If no such query plan can be generated, an error is logged. [#122300][#122300]
+- The [optimizer]({% link {{ page.version.version }}/make-queries-fast.md %}) can now plan constrained scans over partial indexes in more cases, particularly on partial indexes with predicates referencing virtual computed columns. [#123408][#123408]
+
+<h3 id="v23-2-5-operational-changes">Operational changes</h3>
+
+- A minimum Raft scheduler concurrency is now enforced per [store]({% link {{ page.version.version }}/cockroach-start.md %}#storage) so that a node with many stores does not spread workers too thinly. This avoids high scheduler latency across replicas on a store when load is imbalanced. [#120798][#120798]
```

Comment:
```suggestion
- A minimum [Raft]({% link {{ page.version.version }}/2/architecture/replication-layer.md%}#raft) scheduler concurrency is now enforced per [store]({% link {{ page.version.version }}/cockroach-start.md %}#storage) so that a node with many stores does not spread workers too thinly. This avoids high scheduler latency across [replicas]({% link {{ page.version.version }}/architecture/glossary.md %}#replica) on a store when load is imbalanced. [#120798][#120798]
```